e2e/README.md

```diff
@@ -0,0 +1,92 @@
+# Longhorn e2e tests
+
+### Requirement
+
+1. A Kubernetes cluster with 3 worker nodes.
+   - And control node(s) with following taints:
+      - `node-role.kubernetes.io/master=true:NoExecute`
+      - `node-role.kubernetes.io/master=true:NoSchedule` 
+2. Longhorn system has already been successfully deployed in the cluster.
+3. Make sure MountPropagation feature is enabled
+   - For RKE before v0.1.9, you would need the extra parameter feature-gates: `MountPropagation=true` for kube-api and kubelet to enable the feature.
+4. Make sure `nfs-common` or equivalent has been installed on the node to allow the NFS client to work.
```

Comment:
Can just mention running prerequisite scripts on the longhorn/longhorn repo.

Reply:
refined.